### DISCUSSION TITLE
Having a GitHub Action for building .deb and perhaps .rpm packages for our software

### DISCUSSION THREAD

**Discussion by ahoneybun** [cosmic-utils/discussions#1] [Ideas]
I think we would need one per project as I don't think we can have a release for an org so we couldn't just make a cosmic-utils package or something like that.

**Comment by ahoneybun** [comment]
I think using Flatpak would be better like some have already done:

https://flathub.org/apps/dev.edfloreshz.Tasks

**Reply by bhh32** [reply]
Mine actually can't be put in a flatpak because I'm calling Tailscale CLI and OS commands directly. The OS commands I can get around, but I've packaged as a flatpak already and I can't get a direct call to the Tailscale CLI. I've created a RPM and DEB for it already, but automating those builds would be much easier than rebuilding each manually when I get a new feature put in.

**Reply by ahoneybun** [reply]
That's fine this is more for applications that want to be featured in the COSMIC Store as well. It would nice to be able to do it a different way, I guess if system76 were to also package it in their repos it could work.

**Reply by bhh32** [reply]
I'd love to have it featured in the COSMIC store! That would be cool if System76 would give us a package repo for applications/applets like mine that can't be flatpak'd.
